extensions/sql-database-projects/src/models/options/deployOptionsModel.ts

```diff
@@ -8,250 +8,64 @@ import * as mssql from 'mssql';
 export class DeployOptionsModel {
 	public deploymentOptions: mssql.DeploymentOptions;
 
-	public optionsLookup: Record<string, boolean> = {};
-	public optionsMapTable: Record<string, mssql.DacDeployOptionPropertyBoolean> = {};
+	public optionsLookup: Map<string, boolean> = new Map<string, boolean>();
```

Comment:
Can we clean up these names (or, at the very least, just add some comments)? Just looking at the code, it's tough for me to understand why we need optionsLookup and optionsMapTable